### PR TITLE
Jump cloud module 2.1.0

### PR DESCRIPTION
**Pull request by jworkmanjc**
## Issues
* [SA-2938](https://jumpcloud.atlassian.net/browse/SA-2938) - SA-2938-Set-JCSystem-Missing-Description-Field
* [SA-2992](https://jumpcloud.atlassian.net/browse/SA-2992) - Get-JCResults UseBasicParsing
* [SA-2670](https://jumpcloud.atlassian.net/browse/SA-2670) - Improved Error Reporting for Get-JCBackup
* [SA-1375](https://jumpcloud.atlassian.net/browse/SA-1375) - SA-1375-Get-JCObject dynamic search endpoints
* [SA-3009](https://jumpcloud.atlassian.net/browse/SA-3009) - Currently active devices not returning when querying lastContact

### CODE REVIEW

File: PowerShell/JumpCloud Module/Public/RadiusServer/Set-JCRadiusServer.ps1 [hunk 1]
```diff
@@ -20,6 +20,29 @@ Function Set-JCRadiusServer {
         $Results = @()
     }
     Process {
+        # Write-Host "CUSTOM"
```
Comment (jworkmanjc) [inline PowerShell/JumpCloud Module/Public/RadiusServer/Set-JCRadiusServer.ps1:23]: Why is this commented code here?

File: PowerShell/JumpCloud Module/Public/RadiusServer/Set-JCRadiusServer.ps1 [hunk 2]
```diff
@@ -30,9 +53,21 @@ Function Set-JCRadiusServer {
...
+        Write-Host $FunctionParameters.keys
+        Write-Host $FunctionParameters.values
         $Results += Invoke-JCRadiusServer @FunctionParameters
     }
     End {
         Return $Results
     }
-}
\ No newline at end of file
+}
```
Comment (jworkmanjc) [inline PowerShell/JumpCloud Module/Public/RadiusServer/Set-JCRadiusServer.ps1:63]: Why is this here?

File: PowerShell/JumpCloud Module/Public/RadiusServer/Set-JCRadiusServer.ps1 [hunk 3]
```diff
@@ -30,9 +53,21 @@ Function Set-JCRadiusServer {
         ($FunctionParameters).Add('Action', $Action) | Out-Null
         ($FunctionParameters).Add('Type', $Type) | Out-Null
         # Run the command
+        Write-Host $FunctionParameters.keys
```
Comment (jworkmanjc) [inline PowerShell/JumpCloud Module/Public/RadiusServer/Set-JCRadiusServer.ps1:56]: Write-Host? do we really want to be writing this out?

Comment (kmaranionjc) [reply PowerShell/JumpCloud Module/Public/RadiusServer/Set-JCRadiusServer.ps1:56]: Was this used for debugging purposes? If so then we can just remove it

File: PowerShell/JumpCloud Module/Public/RadiusServer/Set-JCRadiusServer.ps1 [hunk 4]
```diff
@@ -30,9 +53,21 @@ Function Set-JCRadiusServer {
...
+# Import-Module "/Users/jworkman/Documents/GitHub/support/PowerShell/JumpCloud Module/JumpCloud.psd1" -force
+# . "/Users/jworkman/Documents/GitHub/support/PowerShell/JumpCloud Module/Public/Authentication/Connect-JCOnline.ps1"
+# . "/Users/jworkman/Documents/GitHub/support/PowerShell/JumpCloud Module/Private/NestedFunctions/Get-JCType.ps1"
+# . "/Users/jworkman/Documents/GitHub/support/PowerShell/JumpCloud Module/Private/NestedFunctions/Out-DebugParameter.ps1"
+# . "/Users/jworkman/Documents/GitHub/support/PowerShell/JumpCloud Module/Private/NestedFunctions/Get-JCColorConfig.ps1"
+# . "/Users/jworkman/Documents/GitHub/support/PowerShell/JumpCloud Module/Private/RadiusServer/Invoke-JCRadiusServer.ps1"
+# . "/Users/jworkman/Documents/GitHub/support/PowerShell/JumpCloud Module/Private/DynamicParameters/Get-JCCommonParameters.ps1"
+# . "/Users/jworkman/Documents/GitHub/support/PowerShell/Deploy/Functions/New-DynamicParameter.ps1"
+# . "/Users/jworkman/Documents/GitHub/support/PowerShell/JumpCloud Module/Private/RadiusServer/Get-DynamicParamRadiusServer.ps1"
+# Set-JCRadiusServer -Id 6359d1751a0a64a598bbf6b5 -networkSourceIp 118.88.96.249 -force
```
Comment (gweinjc) [inline PowerShell/JumpCloud Module/Public/RadiusServer/Set-JCRadiusServer.ps1:73]:
```suggestion
```
Need to get rid of this

File: PowerShell/ModuleChangelog.md [hunk 5]
```diff
@@ -1,25 +1,28 @@
...
 
-N/A
+Get/Set-JCSystem support for description field
 
 #### IMPROVEMENTS:
 
-N/A
+- Description parameter added to Get and Set-JCSystem
+- Added -Force switch parameter that populates New-JCImportTemplate with all headers when user update or new user CSV is created
+- Additional reporting added to Backup-JCOrganization. If failed tasks are detected, the status of the function should report which tasks failed
```
Comment (gweinjc) [inline PowerShell/ModuleChangelog.md:20]:
```suggestion
- Get-JCSystem -filterDateProperty lastContact will now return active systems
```

**Review by gweinjc — CHANGES_REQUESTED** [review]
Need to include SA-3009 into the Issues portion of the PR and include the change in the changelog

**Review by gweinjc — APPROVED** [review]
